firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -69,11 +65,8 @@ extension BrowserViewController {
                                      method: .press,
                                      object: .keyCommand,
                                      extras: ["action": "reload-no-cache"])
-        guard let tab = tabManager.selectedTab else { return }
-
-        if !contentContainer.hasHomepage {
-            tab.reload(bypassCache: true)
-        }
+        store.dispatch(GeneralBrowserAction(windowUUID: windowUUID,
```

Comment:
Same here: Dispatching the action should only happen if the feature flag for toolbar refactor is turned on.